config/_default/menus/menus.zh-cn.toml

```diff
@@ -56,18 +56,18 @@
     weight = 400
 [[footer]]
     parent = "column1"
-    name = "Documentation"
-    url = "/docs/"
+    name = "Download"
+    url = "/docs/install/"
     weight = 410
 [[footer]]
     parent = "column1"
-    name = "Language Guide"
-    url = "/docs/language-guide/"
+    name = "Get started"
```

Comment:
Let's chat through where best to put the language Guide on the Alpha.  We would like it to be discoverable within docs but also have its own URL /language-guide   (not be a subset of docs)